data/json/npcs/talk_tags.json

```diff
@@ -702,7 +717,7 @@
       "<name_b>",
       "<swear> <name_b>",
       "you can fuck right off, you <name_b>",
-      "I've had enough of you, begone.",
+      "I've had enough of you, begone",
```

Comment:
Are the rest of the insults in this auto-capitalized, thus not needing this to be?

Reply:
Honestly I'm ambivalent on this. I just needed short insults which could act as a complete sentence, not as a part of some other sentence. But it could be changed to begin with lowercase letters if really needed.